docs/review_app_setup.sh

```diff
@@ -0,0 +1,95 @@
+#!/bin/bash
+
+# Description: This is a script to help automate the process of building
+# Hokusai review application for Force. It draws upon Artsy's existing review
+# app documentation:
+# https://github.com/artsy/hokusai/blob/master/docs/Review_Apps.md
+# and experiences trying to set up review apps!
+
+echo "[setup_review_app.sh] START"
+
+# Bail out of script on first expression failure and sprint the commands as
+# they are being run.
+set -ev
+
+NAME="$1"
+
+# Generate the Kubernetes YAML needed to provision the application.
+hokusai review_app setup $NAME
+
+# Create the Docker image of your current working direct of Force, and push
+# it to Artsy's docker registry.
+#
+# --force is needed as the current working directory is dirty with (at least)
+# the YAML file generated above.
+# --skip-latest as we're making no claim that this is the "latest" build of the
+# service.
+# --tag to name the image.
+# WARNING: This is likely going to take ~10 mins on your MBP.
+# Be patient and grab some baby carrots.
+hokusai registry push --force --skip-latest --verbose --tag $NAME
+
+# Edit the K8S YAML to reference the proper Docker image
+# sed -i '' 's/:staging/:auct-281/g' hokusai/auct-281.yml
+#
+# The weird `-i ''` is needed on MacOS:
+# https://stackoverflow.com/a/44864004
+# WARNING: This might not work on Linux.
+sed -i '' "s/:staging/:$NAME/g" hokusai/$NAME.yml
+
+# Edit the K8S YAML to remove the instructions that enforce that the service
+# can only be accessible via Cloudflare
+#
+# First, remove the `loadBalancerSourceRanges:` line
+sed -i '' '/loadBalancer/d' hokusai/$NAME.yml
+
+# Then, delete all the IP address lines.
+#
+# WARNING: This is a bit sketchy as this will
+# delete any line of the form "- [number][number][number].". This is my best
+# approx for an regex for an IP address, and I'm sure that there are better
+# ones.
+sed -i '' "/- [[:digit:]][[:digit:]][[:digit:]]./d" hokusai/$NAME.yml
+
+# Provision the review app
+hokusai review_app create $NAME --verbose
+
+# Copy Force staging's ConfigMap to your review app
+hokusai review_app env copy $NAME --verbose
+
+# Copy the staging-shared nginx config to your review app
+hokusai review_app env copy $NAME --configmap nginx-config --verbose
```

Comment:
Ah we may need to also copy over Nginx secrets here until we can build in a Hokusai workaround...

```
kubectl --context staging get secret nginx-secrets -o yaml > nginx-secrets.yml
sed -i '' "s/namespace: default/namespace: $NAME/g" nginx-secrets.yml
kubectl --context staging apply -f nginx-secrets.yml
rm nginx-secrets.yml
```

Reply:
Can you elaborate on the problem here? Running this script as is _seemed_ to build a "working" instance of Force, so I'm not aware of a problem that needs to be solved, but I might be missing something.

Reply:
@izakp Merging this anyways considering that this is working, but happy to iterate from here when you get back on the above.

Reply:
Ah okay maybe the certs are not necessary

Reply:
Actually yes they are if referenced in the k8s config file - see https://artsy.slack.com/archives/C02BC3HEJ/p1566918604140600?thread_ts=1564512635.190400&cid=C02BC3HEJ